redis/rediscache.go

```diff
@@ -0,0 +1,39 @@
+package rediscache
```

Comment:
You don't really need to worry about name collisions like that in go.  Users of the package can always use a different local name.

That said, I agree with @shurcooL that this should be consistent with the other packages in this repo, which use the `foocache` naming style (which conveniently also avoids the concern about naming collision).